backoffice/src/Services/DroitService.cs

```diff
@@ -83,6 +90,7 @@ public async Task<Droit> SaveDroitAsync(Droit droit)
     {
         if ( droit.Id == default )
         {
+            droit.Reference = await GetNextDroitReference();
```

Comment:
This won't affect the reference number, but is it necessary to have this duplicated to AddDroitAsync ?